core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1440,6 +1443,30 @@ private CompiledQuery createTable(final ExecutionModel model, SqlExecutionContex
         }
     }
 
+    private void copyTableReaderMetadataToCreateTableModel(SqlExecutionContext executionContext, CreateTableModel model) throws SqlException {
```

Comment:
We need to check table existence somewhere at the beginning of this method and, if it doesn't exist, we need to throw a `SqlException` to notify the client of the error. Forgot about this during the previous review round, sorry about that.

Also, please add a test to cover this check.

Reply:
I added a test case previously in SqlParserTest but the message returned was'nt user friendly. Now refactored and moved the test case to CreateTableTest. :)